stages/test/test_machine-id.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/python3
+
+import os
+import pathlib
+import unittest.mock
+
+import pytest
+
+import osbuild.meta
+from osbuild.testutil.imports import import_module_from_path
+
+
+def stage(stage_name):
+    test_dir = pathlib.Path(__file__).parent
+    stage_path = pathlib.Path(f"{test_dir}/../org.osbuild.{stage_name}")
+    return import_module_from_path("stage", os.fspath(stage_path))
+
+
+@pytest.fixture(name="machine_id_path")
+def machine_id_path_fixture(tmp_path):
+    machine_id_path = pathlib.Path(f"{tmp_path}/etc/machine-id")
```

Comment:
`pathlib.Path(tmp_path) / "etc/machine-id"` would be the idiomatic way of writing this idiom.

Reply:
Let's do my suggestion as it's slightly more 'sane' than string formatting a path.

Reply:
Sure thing! Thank you, I updated the code now to follow your suggestion.